README.md

```diff
@@ -984,6 +984,10 @@ in inheritance.
   is a hybrid of `Array`'s intuitive inter-operation facilities and
   `Hash`'s fast lookup.
 
+*  Prefer `map` over `collect`, `detect` over `find`, and `reduce` over
```

Comment:
not suggesting we add it, but can someone give me the one-line version of the whys? is it just arbitrarily choosing one? specifically the first 3 (i know that size/length/count actually have behavioral implications)